### PR TITLE
Add get user response dto, add get user by id endpoint

### PR DESCRIPTION
**Pull request by mstruckus**

### CODE REVIEW

File: BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs [hunk 1]
```diff
@@ -43,5 +44,29 @@ public async Task<IActionResult> Register([FromBody] RegisterRequest model)
             return Ok(responseId);
 
         }
+
+        [HttpGet]
+        [Route("profiles/{userId:int}")]
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs]: I missed that in Andrėjus' PR. We should not use `int` as Profile ID for frontend. `int` is for database-generated id, which is incremental and easily brute forceable. We should use "external" ID which is simply random GUID string.

File: BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs [hunk 2]
```diff
@@ -43,5 +44,29 @@ public async Task<IActionResult> Register([FromBody] RegisterRequest model)
...
+                    FirstName = result.FirstName,
+                    LastName = result.LastName,
+                    Email = result.Email,
+                    Office = result.Office
+                };
+                return Ok(response);
+            }
+            catch (NullReferenceException)
+            {
+                return BadRequest($"User with ID={userId} was not found.");
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs]: Have the same notice as for Andrėjus. In my opiniom, it is better not to return human readable errors for the frontend. What we have done - we were using `ErrorCodes` class which has error codes specifically for the frontend, without any special characters. It could look like `employee.employee_not_found`

File: BackendApartmentReservation/BackendApartmentReservation/BackendApartmentReservation.csproj [hunk 3]
```diff
@@ -55,8 +55,4 @@
     </Content>
   </ItemGroup>
 
-  <ItemGroup>
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/BackendApartmentReservation.csproj]: Why is this removed from csproj?

**Review by tomasmuzas — APPROVED** [review]

**Review by andrejus-kuznecovas — APPROVED** [review]
👍 after changes implemented and conflicts solved

File: BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Responses/GetUserResponse.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace BackendApartmentReservation.DataContracts.DataTransferObjects.Responses
+{
+    public class GetUserResponse
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Responses/GetUserResponse.cs]: Rename it to `EmployeeInfo`

File: BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Responses/GetUserResponse.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+        [Required(AllowEmptyStrings = false)]
+        public string LastName { get; set; }
+
+        [Required(AllowEmptyStrings = false)]
+        [EmailAddress]
+        public string Email { get; set; }
+
+        [Required(AllowEmptyStrings = false)]
+        public string Office { get; set; }
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Responses/GetUserResponse.cs]: I will remove this with my PR

File: BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs (hunk 1 above)
Comment (mstruckus) [reply BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs]: Updated with new commit

File: BackendApartmentReservation/BackendApartmentReservation/BackendApartmentReservation.csproj (hunk 3 above)
Comment (mstruckus) [reply BackendApartmentReservation/BackendApartmentReservation/BackendApartmentReservation.csproj]: This must be removed automatically for some reason, added back in

File: BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs (hunk 2 above)
Comment (mstruckus) [reply BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs]: Updated with new commit.

File: BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Responses/GetUserResponse.cs (hunk 4 above)
Comment (mstruckus) [reply BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Responses/GetUserResponse.cs]: Updated with new commit.
